README.md

```diff
@@ -3,6 +3,32 @@
 ## Overview
 
 A lightweight wrapper around [{EpiNow2}](https://github.com/epiforecasts/EpiNow2) to add functionality for deployment in Azure Batch.
+It holds some lightweight helper functions to interface with Azure services, convert input data to EpiNow2's expected input format, and save expected outputs. It also adds metadata and logging.
+
+This package is meant to meet CFA's needs around model deployment in its computational environment.
+The code is open source as part of CFA's goals around development, but it may not be possible to support extensions to additional environments.
+
+## Structure
+
+This repository holds an R package, `{CFAEpiNow2Pipeline}`.
+The repository is structured as a standard R package.
+All PRs pass R CMD check as part of the CI suite as a pre-condition for merge to main.
+
+The package contains contains some adapters and wrappers to run `{EpiNow2}` at moderate computational scale, fitting hundreds or thousands of independent models in parallel with cloud resources.
+The adapters read from datasets with standardized formats and produces outputs as flat files with standard names.
+
+This package does _not_ manage pipeline deployment or kickoff, data extraction and transformation, or model output visualization.
+
+## Components
+
+1. **Config**: Model parameterization and input/output paths are specified by a config, with the path to this config passed at runtime.
+    - The config is validated at runtime, but config generation is specified at pipeline runtime and not part of this package.
+1. **Data**: Data load from the CFA data lake or from a local environment.
+1. **Parameters**: Required parameters are read from the CFA data lake or from a local environment.
+1. **Model run**: The R environment is managed such that the model runs from a fixed random seed, both for EpiNow2 initialization and Stan sampling.
+1. **Outputs**: Model fits are saved as flat files with standardized formats and RDS objects.
```

Comment:
I think it is especially useful for us to have flat files produced by default, as it allows us to use tools like duckdb to very easily read and query remote flat files directly into memory as dataframes.

Reply:
Perhaps eventually we can add a link to the schema of those output tables, but seems a little out of scope for this PR.

Reply:
Stuck that in https://github.com/CDCgov/cfa-epinow2-pipeline/issues/12

Reply:
Ok I spent some time thinking about this feedback:

> This package doesn't implement saveRDS. Model fits being saved is a step in the pipeline, not a contribution to that step by this package. I'd guess the contribution to this step by the package is that it modifys all outputs to be in some standardised format. If that's the bulk of the job being done by the package at this step I think that should be brought out more.

And I don't agree. As I see it, we have had a big computational pipeline that has a bunch of different components: reading the data from an external API, generating parameters like GIs and delays, running EpiNow2, processing EpiNow2 output into various files and formats, and generating figures and summaries for use in production. We've slowly split a number of these components out. The judgement call I'm trying to highlight here is that we want _both_ the RDS object and some particular flat files with some particular files.

That choice maybe sounds obvious, but we've had a few discussions on whether we should only save the RDS object here and do flat file generation downstream. I'm trying to point to our decision that flat file generation is in scope but more general model postprocessing (e.g., scoring) is out of scope.

Reply:
Adam, can you use this additional context to propose a change you find clarifying? I'm not seeing it the same way, so it would be helpful to have a concrete change to discuss.

Reply:
Thank you -- this is super clarifying! 

I'm realizing there's a second confusing thing here; I'm using "pipeline" to refer to both (1) the immediate EpiNow2 model wrapper and (2) the bigger pull data -> fit arbitrary model -> plot/score/publish pipeline of which the EpiNow2 wrapper is a widget to slot into the middle component. I don't have good language to differentiate those.

 I'm going to take another pass at this list with your feedback in mind and see if I can find a middle ground + add a little more clarification.

Reply:
Ok so I took a another crack in https://github.com/CDCgov/cfa-epinow2-pipeline/pull/11/commits/fddaf568bb408f83236bf1cab374767ced73c7ef but I think I mainly just moved things into active voice.

I'm feeling a bit stuck here, so I'm going to resolve this conversation and punt the discussion to be handled via #12 after we have some actual code written we can discuss.